File: Semester-1/Pemrograman-1/Modul-3/C/PRAK304-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>
int main() {
    int input ;
    scanf ("%d",&input);
    if (input == 0){
            printf ("Nol");
    } 
                else if (input <10){ 
            printf ("Satuan");
                }
                   else if (input <20){
            printf ("Belasan");
                   }
                else if (input<99){
            printf ("Puluhan");
                }
        else {
    printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}

File: Semester-1/Pemrograman-1/Modul-3/Python/PRAK304-2310817210014-DamarjatiSuryoLaksono.py
angka = input("")
angka = int(angka)
if (angka == 0) :
    print("Nol")
elif(angka >0 and angka <10) :
    print("Satuan")
elif(angka>=10 and angka <=19):
    print("Belasan")
elif(angka >19 and angka<100):
    print("Puluhan")
else : 
    print("Anda Menginput Melebihi Limit Bilangan")